base.go

```diff
@@ -72,9 +71,14 @@ func (client *rpcClient) call(option callInterface, params ...interface{}) Resul
 	if readerr != nil {
 		panic(readerr)
 	}
-	var res Result
-	json.Unmarshal(body, &res)
-	return res
+	var outerResult Result
+	if err := json.Unmarshal(body, &outerResult); err != nil {
+		panic(err)
+	}
+
+	if err := json.Unmarshal(outerResult.Result, &result); err != nil {
+		panic(err)
+	}
```

Comment:
@cubismic We are unmarshaling twice. The first one unmarshals JSONRPC structure. The second one unmarshals the inner result value.